api-tests/features/inherited-identity/extended-scenarios.feature

```diff
@@ -0,0 +1,290 @@
+@Build
+Feature: Inherited identity extended scenarios
+  Scenario Outline: Successfully migrates <inherited-identity> HRMC profile after partial P2 journey
```

Comment:
This is a slightly odd scenario to test - given how our session store works, the partial journey should have no impact on anything!

Reply:
Yes, thats fine